src/main/java/de/tum/in/www1/artemis/domain/notification/NotificationTarget.java

```diff
@@ -138,6 +140,15 @@ public void setLectureId(Long lectureId) {
         this.lectureId = lectureId;
     }
 
+    @JsonProperty("conversation")
+    public Long getConversationId() {
+        return conversationId;
+    }
+
+    public void setConversationId(Long conversationId) {
+        this.conversationId = conversationId;
+    }
+
     /**
      * @return the NotificationTarget as a JSON String
      *         This is needed to stay consistent with the legacy implementation & data in the DB
```

Comment:
So you suggest to remove the target column from notification table? Did i get it right?

Reply:
I have two issues with this:
1. Usually the JSON gets automatically parsed. However, for some reason we decide to not do it here but to parse it manually with our own objectMapper. So if we configure our ObjectMapper with SpringBoot, we won't have the same configuration here.
2. It talks about legacy implementation because the data in the database couldn't be updated. This was added 23.12.2021. The same date I introduced the Java Migration. So the developer probably didn't know about that option yet.

Reply:
I see, i will definitely check this in my follow-up. Lets leave this discussion open.